package.json

```diff
@@ -22,7 +22,8 @@
     "build:3": "chmod +x ./build/main.js",
     "prepackage": "rimraf build cjs dist && npm run build && npm run rollup",
     "test": "vitest run --test-timeout=10000",
-    "test:watch": "vitest --test-timeout=10000"
+    "test:watch": "vitest --test-timeout=10000",
+    "version": "node ./scripts/bump.js && git add src/version.ts"
```

Comment:
I wonder if we need to update the README to include this step; or does this script run automatically when you follow the instruction of the releasing documentation?

Reply:
It runs automatically as a part of `npm version A.B.C` which is a step in release process already - so nothing changes in how the release process should be done.